source/slang/slang-ir-glsl-legalize.cpp

```diff
@@ -2993,14 +2993,6 @@ void legalizeEntryPointsForGLSL(
     {
         legalizeEntryPointForGLSL(session, module, func, context, glslExtensionTracker);
     }
-
-    // GLSL should NEVER mangle user declared ray locations; else user setting a location to reference  
-    // (GLSL references locations by a constexpr number) could be invalid. Furthermore, we need to fetch
-    // types and what location variable is where when we resolve locations to translate GLSL->SPIR-V/HLSL
-    if (!context->getTargetProgram()->getOptionSet().getBoolOption(CompilerOptionName::AllowGLSL)) 
-    {
-        assignRayPayloadHitObjectAttributeLocations(module);
```

Comment:
We don't need this anymore?

Reply:
we need it, I was just confused on why things were working without it (turns out, visual studio was just not working properly, so I rebuilt everything); I will adjust the call though